docker/pypi/wmagent-mariadb/Dockerfile

```diff
@@ -0,0 +1,81 @@
+ARG TAG=11.0
+ARG MDB_TAG=$TAG
+FROM mariadb:${MDB_TAG}
+MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+
+ARG MDB_TAG=$TAG
+ENV MDB_TAG=${MDB_TAG}
+RUN echo MDB_TAG=$MDB_TAG
+
+RUN apt-get update && apt-get install -y vim less sudo wget unzip python3 pip
+
+# Install some debugging tools
+RUN apt-get install -y hostname net-tools iputils-ping procps emacs-nox tcpdump && apt-get clean
```

Comment:
you may combine this line with previous out for apt-get to reduce number of steps.